src/ImageSharp/Processing/Processors/Transforms/Resize/ResizeWorker.cs

```diff
@@ -184,13 +186,13 @@ private void CalculateFirstPassValues(RowInterval calculationInterval)
                 // Span<Vector4> firstPassSpan = transposedFirstPassBufferSpan.Slice(y - this.currentWindow.Min);
                 ref Vector4 firstPassBaseRef = ref transposedFirstPassBufferSpan[y - this.currentWindow.Min];
 
-                for (int x = this.targetWorkingRect.Left; x < this.targetWorkingRect.Right; x++)
+                for (nint x = left; x < right; x++)
```

Comment:
Same here:
```suggestion
                for (nint x = 0; x < (right - left); x++)
```
? 
And adjust below to just `x` instead of `x - left`.

Reply:
Ah, I have to add an extra incremental here. I need both `x` & `x - left`.